packages/lab/src/typography/Text.css

```diff
@@ -113,38 +108,34 @@ h4.uitkText small,
 label.uitkText,
 .uitkText-label {
   font-size: var(--uitk-text-caption-fontSize);
-  line-height: var(--uitk-text-caption-lineHeight);
+  line-height: var(--uitk-text-lineHeight-backwardsCompat, var(--uitk-text-caption-lineHeight));
 }
 
 /* Help text */
 .uitkText-helpText.uitkText {
   font-style: var(--uitk-text-help-fontStyle);
   color: var(--uitk-text-secondary-foreground);
   font-size: var(--uitk-text-help-fontSize);
-  line-height: var(--uitk-text-help-lineHeight);
+  line-height: var(--uitk-text-lineHeight-backwardsCompat, var(--uitk-text-help-lineHeight));
 }
 
 /* Figure 1 */
 .uitkText-figure1.uitkText {
   font-size: var(--uitk-text-figure1-fontSize);
-  line-height: var(--uitk-text-figure1-lineHeight);
+  line-height: var(--uitk-text-lineHeight-backwardsCompat, var(--uitk-text-figure1-lineHeight));
   font-weight: var(--uitk-text-figure1-fontWeight);
 }
 
 /* Figure 2 */
 .uitkText-figure2.uitkText {
   font-size: var(--uitk-text-figure2-fontSize);
-  line-height: var(--uitk-text-figure2-lineHeight);
+  line-height: var(--uitk-text-lineHeight-backwardsCompat, var(--uitk-text-figure2-lineHeight));
   font-weight: var(--uitk-text-figure2-fontWeight);
 }
 
 /* Figure 3 */
 .uitkText-figure3.uitkText {
   font-size: var(--uitk-text-figure3-fontSize);
-  line-height: var(--uitk-text-figure3-lineHeight);
+  line-height: var(--uitk-text-lineHeight-backwardsCompat, var(--uitk-text-figure3-lineHeight));
```

Comment:
we shouldn't do backwards compat like this, we should be providing --uitkText-lineHeight as CSS API and setting this in a .backwardsCompat class if it's needed

Reply:
+ in any case CSS API should be --uitk<ComponentName>, --uitk- prefix is usually kept for characteristic/foundation prefix

Reply:
see https://github.com/jpmorganchase/uitk/pull/260 for more changes on this